terraform/cp4ba/main.tf

```diff
@@ -9,68 +9,74 @@ data "ibm_resource_group" "group" {
   name = var.resource_group
 }
 
-resource "null_resource" "mkdir_kubeconfig_dir" {
-  triggers = { always_run = timestamp() }
-  provisioner "local-exec" {
-    command = "mkdir -p ${var.cluster_config_path}"
-  }
-}
-
 module "create_cluster" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/roks"
-
+//  source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks"
+  source = "../../../terraform-ibm-cloud-pak/modules/roks"
```

Comment:
This needs to be reverted back to using the github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks

Reply:
fixed